src/test/java/io/codiga/plugins/jetbrains/testutils/Constants.java

```diff
@@ -0,0 +1,20 @@
+package io.codiga.plugins.jetbrains.testutils;
+
+import com.google.common.collect.ImmutableList;
+import io.codiga.plugins.jetbrains.annotators.Annotation;
+
+public class Constants {
+    public final static String recipe_sample = "dGhyZWFkOjpzcGF3bihtb3ZlIHx8IHsKICAvLyB0aHJlYWQgY29kZSBoZXJlCiAgNDIKfSk7";
+
+    public final static String ClipboardContent = "$ClipboardContent$";
+    public final static String FileDir = "$FileDir$";
+    public final static String FileName = "$FileName$";
+    public final static String FileNameWithoutExtension = "$FileNameWithoutExtension$";
+    public final static String FilePath = "$FilePath$";
+    public final static String FileRelativePath = "$FileRelativePath$";
+    public final static String LineNumber = "$LineNumber$";
+    public final static String ProjectFileDir = "$ProjectFileDir$";
+    public final static String ProjectName = "$ProjectName$";
+    public final static String SelectedText = "$SelectedText$";
```

Comment:
All constants in Java must be `UPPER_CASE` - see https://explainjava.com/java-constants/